huntsman/utils/pyro/__init__.py

```diff
@@ -62,20 +62,20 @@ def run_camera_server(ignore_local=False, unmount_sshfs=True, logger=None,
         $HUNTSMAN_POCS/conf_files/pyro_camera.yaml only. If False will allow
         $HUNTSMAN_POCS/conf_files/pyro_camera_local.yaml to override the default configuration.
         Default False.
-        
-        unmount_sshfs (bool, optional): If True, unmounts the sshfs upon 
+
+        unmount_sshfs (bool, optional): If True, unmounts the sshfs upon
             termination of the camera server.
-            
+
     """
     if logger is None:
         logger = DummyLogger()
-        
-    #Mount the SSHFS
-    mountpoint = sshfs_mount.mount_sshfs()
-    
+
+    #Mount the SSHFS images directory
+    mountpoint = sshfs_mount.mount_images_dir()
```

Comment:
Looks like you should be passing the logger through to most of these.

```suggestion
    mountpoint = sshfs_mount.mount_images_dir(logger=logger)
```